source/kra/layer.d

```diff
@@ -147,9 +114,13 @@ public:
 }
 
 /**
-    A Krita Layer
+    Encompasses all layers, including masks
+
+    Unused params:
```

Comment:
DDOC Sections can not have spaces; in this case `Unused_Parameters` might be more appropriate.